README.md

```diff
@@ -20,7 +20,7 @@ through setting up and using TensorBoard. There's an associated
 
 [this video tutorial]: https://www.youtube.com/watch?v=eBbEDRsCmv4
 
-[tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/dandelionmane/tf-dev-summit-tensorboard-tutorial
+[tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/decentralion/tf-dev-summit-tensorboard-tutorial
```

Comment:
Do we want to move this repository into the @tensorflow organization?

Reply:
That would be great; that way it can actually be maintained.
Do you want me to just transfer the repository to @tensorflow?

Reply:
Yes; that’d be great. Thanks!

Reply:
Please leave a comment here when you initiate the transfer.

Reply:
Yes, add me as admin to the repo and I can transfer it.

Reply:
Sent you an invite link.

Reply:
Accepted. Still need admin for the repo, then I can transfer it, I think.

Reply:
Retaining `martinwicke/tf-dev-summit-tensorboard-tutorial`.

(Googlers, see <http://b/120805989>.)